src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -91,21 +91,6 @@ def test_basic_evaluation(self, pf: PFClient, runtime: str, randstr: Callable[[s
         assert isinstance(eval_run, Run)
         pf.runs.stream(run=eval_run.name)
 
-        # evaluation run without data
```

Comment:
why delete this?

Reply:
as offline synced, I will update this test in another PR (will before this PR)